vision/unit_tests/test_module.py

```diff
@@ -273,5 +273,99 @@ def test_return(self):
+        """
+        IMAGE_SIZE = [1920, 1080]
+
+        # testing with ndarray of all zeroes, arbitrary depth value,
+        #   and arbitrary center value
+        depth_image = np.zeros(IMAGE_SIZE, dtype=int)
+        depth_val = 300.0
+        center = (300, 300)
+        roi = region_of_interest(depth_image, depth_val, center)
+
+        self.assertIs(type(depth_image), np.ndarray)
+        self.assertIs(type(depth_val), float)
+        self.assertIs(type(center), tuple)
+        self.assertIs(type(roi), np.ndarray)
+
+    def test_region_of_interest(self):
+        img_dir = os.path.join(gparent_dir, "vision_images/module/Feb29")
+        files = os.listdir(img_dir)
+
+        # hand-picked images for unit test, only depth image is necessary
+        estimates = {
+            # FORMAT: "name_of_file_without_color-depth_tail" : [estimated_center (x, y), estimated_size{x,y)]
+            "2020-02-29_15.48.10.480039": [(990, 600), (150, 200)],
+            "2020-02-29_15.35.41.406247": [(830, 690), (120, 150)],
+            "2020-02-29_15.35.57.551331": [(1228, 653), (90, 120)],
+            "2020-02-29_15.35.56.436446": [(1170, 644), (145, 180)],
+            "2020-02-29_15.40.42.462549": [(948, 620), (60, 70)]
+            # NOTE: The center tuples are estimated because center function does not work at the moment
+        }
+
+        for current_file in estimates.keys():
+            # grabs and loads depth file (assumes all files are in order of color1, depth1, color2, depth2, etc.)
+            current_depth_file = (
+                os.path.join(img_dir, current_file) + "-depthImage.npy"
+            )
+            current_depth_image = np.load(current_depth_file)
+
+            # get center
+            current_center = estimates[current_file][0]
+
+            # get region of interest
+            roi = region_of_interest(
+                current_depth_image,
+                current_depth_image[current_center[1]][current_center[0]],
+                current_center,
+            )
+
+            estimated_height = estimates[current_file][1][1]
+            estimated_width = estimates[current_file][1][0]
+            calculated_height = roi.shape[0]
+            calculated_width = roi.shape[1]
+
+            self.assertTrue(
+                estimated_height * 0.8 <= calculated_height <= estimated_height * 1.2
+            )  # within ±20%
+            self.assertTrue(
+                estimated_width * 0.8 <= calculated_width <= estimated_width * 1.2
+            )  # within ±20%
+
+    def test_return(self):
+        """
+        Verify returns only expected output
+
+        Returns
+        -------
+        ndarray
+        """
+        IMAGE_SIZE = [1920, 1080]
+
+        # testing with ndarray of all zeroes, arbitrary depth value,
+        #   and arbitrary center value
+        depth_image = np.zeros(IMAGE_SIZE, dtype=int)
+        roi = region_of_interest(depth_image, 300, (300, 300))
```

Comment:
`roi = region_of_interest(depth_image, 300, (300, 300))`

The 300s here should be variables so it is clear what is being tested.